### PR TITLE
change detection method of process elevation

### PR DESCRIPTION
**Pull request by OrbbQ3**

### CODE REVIEW

**Comment by OrbbQ3** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by OrbbQ3** [comment]
so, hope now all is ok :)

File: SystemInformer/tokprp.c [hunk 1]
```diff
@@ -437,19 +437,25 @@ PWSTR PhGetPrivilegeAttributesString(
     }
 }
 
-PWSTR PhGetElevationTypeString(
+PPH_STRING PhGetElevationTypeString(
+    _In_ BOOLEAN isElevated,
```
Comment (dmex) [inline SystemInformer/tokprp.c:441]: Caps `IsElevated`

File: SystemInformer/tokprp.c [hunk 2]
```diff
@@ -437,19 +437,25 @@ PWSTR PhGetPrivilegeAttributesString(
...
     _In_ TOKEN_ELEVATION_TYPE ElevationType
-    )
+)
```
Comment (dmex) [inline SystemInformer/tokprp.c:443]: Indent `)`

**Review by dmex — APPROVED** [review]

File: SystemInformer/tokprp.c (hunk 1 above)
Comment (OrbbQ3) [reply SystemInformer/tokprp.c]: done

File: SystemInformer/tokprp.c (hunk 2 above)
Comment (OrbbQ3) [reply SystemInformer/tokprp.c]: done

**Pull request merged**